examples/basic_selectmenu.py

```diff
@@ -0,0 +1,50 @@
+import discord
+from discord.ext import commands
+
+client = commands.Bot(command_prefix='?')   #define our bot
```

Comment:
Because this isn't explicitly an `ext,commands` example, you should be using `discord.Client` for the example. Use an `on_message` listener. Doesn't make for the best bots, but that's why `commands` is an extension rather than the default.

Reply:
in other exaples is discord ext used too? https://github.com/Voxel-Fox-Ltd/Novus/blob/master/examples/basic_bot.py

Reply:
The example you've linked is specifically _for_ bots, but I will concede that other examples also use the commands ext. I'll leave as is and fix them all later.